src/plugins/intel_cpu/src/nodes/detection_output.cpp

```diff
@@ -289,9 +288,16 @@ void DetectionOutput::execute(dnnl::stream strm) {
             });
         } else {
             // MXNet style
-            int *pbuffer = indicesBufData + n * classesNum * priorsNum;
+            int offImg = n * priorsNum * classesNum;
```

Comment:
minor
```suggestion
            const int offImg = n * priorsNum * classesNum;
```
You can add `const` for all variables which are constant

Reply:
updated, thanks Alexandra.